syntax/tests/grammar.rs

```diff
@@ -0,0 +1,7 @@
+#[test]
+fn parse_abnf_grammar() {
+	abnf::rulelist(concat!(
+		include_str!("../src/grammar.abnf"),
+		include_str!("../src/grammar-rfc3987.abnf")
+	)).unwrap();
```

Comment:
maybe it could also validate every `.tldr` file under `/tests`?

Reply:
The `abnf` crate doesn't even parse the grammar properly. I think it doesn't implement the update [RFC 7405](https://www.rfc-editor.org/rfc/rfc7405) on case sensitive strings. And I think it's just a parser, it cannot generate a parser.